test/test_httpclient.rb

```diff
@@ -829,16 +845,6 @@ def test_get_with_block_arity_2_and_redirects
     assert_nil(res.content)
   end
 
-  def test_get_with_block_string_recycle
```

Comment:
I feel bad about removing that test, but unfortunately `read_block_size` doesn't make any sense in case `sendfile()` is used.

But the next test, test the same behavior with chunked response, so I think it's okish to remove it.